src/ComputeSharp.D2D1/Attributes/Enums/D2D1PixelOption.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ComputeSharp.D2D1;
+
+/// <summary>
+/// Indicates how the sampling of a D2D1 pixel shader will be restricted. This indicates whether the vertex buffer
+/// is large and tends to change infrequently or smaller and changes frequently (typically frame over frame). 
+/// </summary>
+/// <remarks>
+/// This type exposes the available values in <see href="https://docs.microsoft.com/windows/win32/api/d2d1effectauthor/ne-d2d1effectauthor-d2d1_pixel_options"/>.
+/// </remarks>
+public enum D2D1PixelOption
```

Comment:
should be plural, `D2D1PixelOptions`. Not just because that's what it's called in D2D, but because all the other places say option_s_

Reply:
This should probably be marked with `[Flags]` as well?

Reply:
Fixed in ca05ea2a5fb90a27ffd91dcbd3f4326d321fe0a2, also added `[Flags]`.